src/main/scala/org/labrad/manager/Handlers.scala

```diff
@@ -398,68 +440,119 @@ class ManagerImpl(id: Long, name: String, hub: Hub, stub: ManagerSupport, tracke
+               |
+               |Can be called with two arguments specifying the message id to use for context
+               |expiration notifications and a flag indicating whether to send combined "expire all"
+               |messages. Or can be called with no arguments to disable sending context expiration
+               |messages.
+               |
+               |Context expiration messages will be sent to the given message id in the context in
+               |which this setting was called, for all contexts which have been used to make
+               |requests to this server. Expiration can be triggered either by someone explicitly
+               |calling one of the settings "Expire Context" or "Expire All", or by the connection
+               |with matching high id being closed. In the case of an individual context expiration,
+               |the message data will be a context id, given as a cluster of (high word, low word).
+               |If the expire all flag is false, then when a connection closes or "Expire All" is
+               |called the manager will also send out expiration messages with a context id cluster
+               |for all associated contexts. However, if the expire all flag was true, then we
+               |instead will send one single message containing just the high word of the context
+               |(which matches the connection id).""")
   def notifyOnContextExpiration(r: RequestContext, data: Option[(Long, Boolean)]): Unit = data match {
     case Some((msgId, expireAll)) => stub.startNotifications(r, msgId, expireAll)
     case None => stub.stopNotifications(r)
   }
 
-  @Setting(id=120, name="S: Start Serving", doc="(Servers only) Signal that the server is ready to receive requests. Before this point, the server will not appear in the list of active servers nor any metadata lookups.")
+  @Setting(id = 120,
+           name = "S: Start Serving",
+           doc = """(Servers only) Signal that the server is ready to receive requests.
+               |
+               |Before calling this setting, the server will not appear in the list of active
+               |servers nor any metadata lookups.""")
   def startServing(): Unit = {
     stub.startServing
     messager.broadcast(Manager.ConnectServer(id, name), sourceId = Manager.ID)
   }
 
   // utility methods (should stay local)
 
-  @Setting(id=200, name="Data To String", doc="Convert data into its human-readable string representation.")
+  @Setting(id = 200,
+           name = "Data To String",
+           doc = """Convert data into its human-readable string representation.""")
   def dataToString(data: Data): String = data.toString
 
-  @Setting(id=201, name="String To Data", doc="Parse a string into labrad data.")
+  @Setting(id = 201,
+           name = "String To Data",
+           doc = """Parse a string from human-readable representation into labrad data.""")
   def stringToData(str: String): Data = Data.parse(str)
```

Comment:
I think we should deprecate this -- or at least state what we expect about the stability of the string representation.